ci/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:22.04@sha256:a6d2b38300ce017add71440577d5b0a90460d0e57fd7aec21dd0d1b0761bbfb2
```

Comment:
I know dependabot is able to update pinned dependencies on github actions and expect it will be able to manage docker images too. I will test it on a fork and confirm if it can do it.

Reply:
For future updates dependabot will upgrade the pinned docker image. I will do it here and close dependabot pending PR.
 
![image](https://github.com/codeplaysoftware/SYCL-samples/assets/2277170/533b3684-fa62-4dcb-a3f6-75557f3d4b7a)